config/database.php

```diff
@@ -56,13 +53,12 @@
 
         'mysql' => [
             'driver' => 'mysql',
-            'url' => env('DATABASE_URL'),
-            'host' => env('DB_HOST', $fallback_db_config['db_host']),
-            'port' => env('DB_PORT', $fallback_db_config['db_port']),
-            'database' => env('DB_DATABASE', $fallback_db_config['db_name']),
-            'username' => env('DB_USERNAME', $fallback_db_config['db_user']),
-            'password' => env('DB_PASSWORD', $fallback_db_config['db_pass']),
-            'unix_socket' => env('DB_SOCKET', $fallback_db_config['db_socket']),
+            'host' => env('DB_HOST', 'localhost'),
+            'port' => env('DB_PORT', ''),
```

Comment:
```suggestion
            'port' => env('DB_PORT', '3306'),
```

Reply:
Should be handled by the underlying mysql library, but that is what upstream has, so should be fine.